### DISCUSSION TITLE
Criar uma coluna nova na lista de vendas

### DISCUSSION THREAD

**Discussion by julianoaraujonk** [RamonSilva20/mapos#1813] [General]
olá, gostaria de saber como faço para na tela de lista de vendas colocar uma coluna nova com o nome do vendedor, só estou conseguindo pegar o id do usuário queria pegar o nome do usuário.

**Comment by Pr3d4dor** [comment]
@julianoaraujonk O que vc tentou fazer até agora?

**Reply by julianoaraujonk** [reply]
consegui resolver ,  agora está mostrando, não estava fazendo o join na tabela usuário para pegar o nome do usuário.

Nº | Data | Cliente | Faturado | Vendedor | Ações
